__tests__/app.test.js

```diff
@@ -179,10 +179,38 @@ describe("GET /api/articles", () => {
               body: expect.any(String),
               created_at: expect.any(String),
               votes: expect.any(Number),
-              comment_count: expect.any(String),
+              comment_count: expect.any(Number),
             })
           );
         });
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("Responds with an array of comments for the given article_id", () => {
```

Comment:
What about comments for an article id that doesn't exist or is of the wrong type?